src/ProjectService.Data/UserRepository.cs

```diff
@@ -48,9 +67,23 @@ public void AddUsersToProject(IEnumerable<DbProjectUser> dbProjectUsers , Guid p
             _provider.ProjectsUsers.AddRange(dbProjectUsers);
         }
 
-        public IEnumerable<DbProjectUser> Find(Guid userId)
+        public IEnumerable<DbProjectUser> Get(Guid userId)
+        {
+            return Find(new FindDbProjectsUserFilter { UserId = userId });
+        }
```

Comment:
This need use FirstOrDefault()

Reply:
The method has bad name, i rename to Find